Fakebook.Profile/Fakebook.Profile.DataAccess/EntityModel/EntityProfile.cs

```diff
@@ -1,50 +1,123 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
+using Fakebook.Profile.Domain.Utility;
 
 namespace Fakebook.Profile.DataAccess.EntityModel
 {
     /// <summary>
     /// Database model of a profile.
     /// </summary>
+    [Table("Profile")]
     public class EntityProfile
     {
+        #nullable enable
```

Comment:
is it a well-supported pattern to use nullable reference types, but only for the ef model classes, like this? does it work correctly?